README.md

```diff
@@ -59,6 +61,13 @@ _Default: no timeout (other timeouts will eventually close the connection)._
 Sets timeout (in seconds) for establishing TCP connection to target website. Affects all requests.  
 _Default: 20 seconds._
 
+- **upstream [https://username:password@upstreamproxy.site:443]**  
+Sets upstream proxy to route all requests through it. For probe resistance reasons,
+upstreaming proxy will still serve its' website, unless correct credentials are provided.  
+Supported schemes: https, http, socks5. **WARNING**: http and socks5 leak everything in plaintext,
+including requests made and upstream proxy credentials – don't do it. We may decide to
+disable insecure schemes in future.  
```

Comment:
Maybe I should just disable it right now?

Reply:
Yeah, don't do insecure. :) I second that.